src/Sidekick.Apis.Poe/Parser/Properties/PropertyParser.cs

```diff
@@ -63,6 +63,7 @@ public async Task Initialize()
             new MonsterPackSizeProperty(gameLanguageProvider),
             new BlightedProperty(gameLanguageProvider),
             new BlightRavagedProperty(gameLanguageProvider),
+            new LastTwelveHoursProperty(gameLanguageProvider),
```

Comment:
This is not how I want this to be done as properties are part of the item, not trade filters.